plugins/CoreHome/lang/de.json

```diff
@@ -29,7 +29,7 @@
         "MakeOneTimeDonation": "Stattdessen einmalig spenden.",
         "Menu": "Menü",
         "NoPrivilegesAskPiwikAdmin": "Sie sind angemeldet als '%1$s', aber es wurden keine Berechtigungen für diesen Benutzer in Piwik gesetzt. %2$s Bitte fragen Sie Ihren Piwik Administrator (klicken für E-Mail)%3$s um Zugriff auf die Webseite zu erhalten.",
-        "OnlyForSuperUserAccess": "Dieses Widget wird nur Benutzern mit Hauptadministrator-Berechtigung angezeigt.",
+        "OnlyForSuperUserAccess": "Dieses Widget wird im standart Dashboard nur Benutzern mit Hauptadministrator-Berechtigung angezeigt.",
```

Comment:
I think the "Duden" says "standard", maybe use "Standard-Dashboard" as we have used this wording in other translations so far